src/compiler/scala/tools/nsc/tasty/bridge/SymbolOps.scala

```diff
@@ -74,7 +74,7 @@ trait SymbolOps { self: TastyUniverse =>
     def isTraitParamAccessor: Boolean = sym.owner.isTrait && repr.tflags.is(FieldAccessor|ParamSetter)
 
     def isParamGetter: Boolean =
-      sym.isMethod && sym.repr.tflags.is(FlagSets.ParamGetter)
+      sym.isMethod && /*sym.*/repr.tflags.is(FlagSets.ParamGetter)
```

Comment:
intentional to leave this?

Reply:
`sym.repr` is "expressive" but wraps instead of just calling the method. I left it to show intent. The decorator can't be anyval. But if you think it looks like a mistake, I'll zap it.

Reply:
that was one possible interpretation I considered, but I think on the balance it's more confusing than helpful. especially since there's a `sym.`-less `repr` just 3 lines above